src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -596,6 +596,21 @@ protected DynamoModel(IStartConfiguration config)
                 PreferenceSettings.PropertyChanged += PreferenceSettings_PropertyChanged;
             }
 
+            UpdateManager = config.UpdateManager ?? new DefaultUpdateManager(null);
+
+            var hostUpdateManager = config.UpdateManager;
+
+            if (hostUpdateManager != null)
+            {
+                HostName = hostUpdateManager.HostName;
+                HostVersion = hostUpdateManager.HostVersion == null ? null : hostUpdateManager.HostVersion.ToString();
+            }
+            else
+            {
+                HostName = string.Empty;
+                HostVersion = null;
+            }
+
```

Comment:
This code is pretty independent of everything else...so it looks to be safe to move upstream.

Reply:
@pinzart Agree